### ISSUE TITLE
[Bug] Graph settings in "widget wrappers for multiple widgets" (e.g. Visits Overview (with graph)) are not being persisted

### ISSUE THREAD

**Issue by peterbo** [piwik/piwik#21944]
### What happened?

Graph settings in "widget wrappers for multiple widgets" (e.g. Visits Overview (with graph)) are not being persisted. 

Looks similar but was fixed in the past: https://github.com/matomo-org/matomo/issues/11499 

### What should happen?

changes to the graphs should be persisted

### How can this be reproduced?

reproducible on demo:
- add widget Visits Overview (with graph) to the dashboard
- add other metrics to be plotted
- reload dashboard

### Matomo version

5.x

### PHP version

all

### Server operating system

all

### Computer operating system

all

### Validations

- [X] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [X] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.

**Comment by mneudert** [comment]
Hey @peterbo,

I was able to reproduce that problem with the `Ecommerce > Overview` and the `Goals > [specific goal]` widgets. Not the `Visits over time` widget, though :thinking: 

The first two widgets mentioned send a `Dashboard.saveLayout` when added, but not when the metrics are changed. In that step a `CoreHome.saveViewDataTableParameters` request is sent.

When reloading the dashboard in 5.x these parameters are not used, in 4.x they seem to be.

**Comment by peterbo** [comment]
> Not the Visits over time widget, though 🤔

That's the wrong one. Try the "Visits Overview (with graph)" one (which is the combination widget)

**Comment by mneudert** [comment]
You're right :facepalm: That one is also not using the saved options as you mentioned.
